test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -61,6 +61,11 @@ def to_numpy(elem):
         if elem.requires_grad:
             return elem.detach().cpu().numpy()
         else:
+            # convert from torch.bfloat16 to numpy.half because
+            # numpy does not support bfloat16.
+            # Tests need to use higher tolerance in this scenario
+            if elem.dtype == torch.bfloat16:
```

Comment:
Any idea why inputs of type float16 and float creates output of type bfloat16?

Reply:
Probably because bfloat16 has a higher range than float16 and it is m=faster to convert float32 to bfloat16

https://nhigham.com/2018/12/03/half-precision-arithmetic-fp16-versus-bfloat16/ has a nice write up